documentation/source/release_notes.rst

```diff
@@ -4,6 +4,41 @@ Release Notes
 
 All releases are available from the `GitHub Releases Page <https://github.com/cocotb/cocotb/releases>`_.
 
+cocotb 1.3
+==========
+
+Released on 2 January 2020.
+
+This will likely be the last release to support Python 2.7.
+
+New features
+------------
+- :func:`cocotb.triggers.with_timeout`, for a shorthand for waiting for a trigger with a timeout. (:pr:`1119`)
+- New ``timeout_time`` and ``timeout_unit`` arguments to :func:`cocotb.test`, for adding test timeouts. (:pr:`1119`)
+- The ``expect_error`` argument to :func:`cocotb.test` now accepts a specific exception type. (:pr:`1116`)
+- New environment variable :envvar:`COCOTB_RESULTS_FILE`, to allow configuration of the xunit XML output filename.  (:pr:`1053`)
+- New makefile variables :make:var:`COCOTB_HDL_TIMEUNIT` and :make:var:`COCOTB_HDL_TIMEPRECISION` for setting the default time unit and precision that should be assumed for simulation when not specified by modules in the design. (:pr:`1113`)
+- Support for the :ref:`Verilator` simulator (version 4.020 and above).
+- A new ``bus_separator`` argument to :class:`cocotb.drivers.BusDriver`. (:pr:`1160`)
+- A new ``start_high`` argument to :meth:`cocotb.clock.Clock.start`. (:pr:`1036`)
+- A new :data:`cocotb.__version__` constant, which contains the version number of the running cocotb. (:pr:`1196`)
+
+Notable changes and bug fixes
+-----------------------------
+- ``DeprecationWarning``\ s are now shown in the output by default.
+- Tracebacks are now preserved correctly for exceptions in Python 2. The tracebacks in all python versions are now a little shorter.
+- :func:`cocotb.external` and :func:`cocotb.function` now work more reliably and with fewer race conditions.
+- A failing ``assert`` will be considered a test failure. Previously, it was considered a test *error*.
+- :meth:`~cocotb.handle.NonConstantObject.drivers` and :meth:`~cocotb.handle.NonConstantObject.loads` now work correctly in Python 3.7 onwards.
```

Comment:
Does this mean "now also work correctly in Python 3.7 onwards" or "now work correctly in Python 3.7 onwards (but are incorrect in lower versions)."?

Reply:
No, simply that they _only_ worked in lower version before.